### PR TITLE
Update `elasticsearch` to version `7.16.3`

### PR DESCRIPTION
**Pull request by xRuiAlves**

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-ThisBuild / version := "0.18.7-SNAPSHOT"
+ThisBuild / version := "0.18.7"
```
Comment (dimamo5) [inline version.sbt:1]: I don't think we need to change the version here. This will be automatically updated with the release of a new version

**Review by dimamo5 — CHANGES_REQUESTED** [review]

File: version.sbt (hunk 1 above)
Comment (xRuiAlves) [reply version.sbt]: @dimamo5 yeap, you're right! Reverted the version updating commit, should be good now 👍

**Review by dimamo5 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,13 @@
 
 ### Security
 
+## [0.18.7] - 2023-02-24
+
+### Changed
+- Update elasticsearch-related dependencies to `7.13.x` ([#432](https://github.com/adzerk/apso/pull/432)).
```
[the remark this answers is not in the document]
Comment (xRuiAlves) [reply CHANGELOG.md]: Oops, my bad! Fixed 👍

**Review by dimamo5 — DISMISSED** [review]
Thanks

**Review by dimamo5 — APPROVED** [review]

**Pull request merged**